src/core/ad.jl

```diff
@@ -4,12 +4,13 @@
 using Bijectors
```

Comment:
Can this be removed now that Bijectors is loaded in src/core/Core.jl?

Reply:
Sure